### PR TITLE
JanDeDobbeleer.OhMyPosh version 3.166.1

### PR DESCRIPTION
**Pull request by JanDeDobbeleer**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18690)

### CODE REVIEW

File: manifests/j/JanDeDobbeleer/OhMyPosh/3.166.1/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: JanDeDobbeleer.OhMyPosh
+PackageVersion: 3.166.1
+InstallModes:
+- silent
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/j/JanDeDobbeleer/OhMyPosh/3.166.1/JanDeDobbeleer.OhMyPosh.installer.yaml:9]:
```suggestion
- Architecture: x64
```
You didn't do what ryfu-msft suggested https://github.com/microsoft/winget-pkgs/pull/18482#issuecomment-866415557

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/18482#issuecomment-866415557

**Comment by OfficialEsco** [comment]
Bro you don't have to close the PR to make the changes >.<

**Comment by denelon** [comment]
@OfficialEsco, I believe he is working on automation from his repository so he wants to get it "right" from the source so he can walk away...

**Comment by JanDeDobbeleer** [comment]
@OfficialEsco indeed what @denelon said. I'm not doing the change here as the PR is automated from my own codebase. I basically forgot to do the change yesterday, I'm on it ;-)

File: manifests/j/JanDeDobbeleer/OhMyPosh/3.166.1/JanDeDobbeleer.OhMyPosh.installer.yaml (hunk 1 above)
Comment (JanDeDobbeleer) [reply manifests/j/JanDeDobbeleer/OhMyPosh/3.166.1/JanDeDobbeleer.OhMyPosh.installer.yaml:9]: I know, I forgot to adjust that. These PR's are automated from my codebase. The weird thing remains that this did work before and no changes were made to the definition, nor the Inno installer. So, it's not really an explanation.

**Pull request closed without merging**